docs-src/clusters/how-to-set-up-visibility-in-a-temporal-cluster.md

```diff
@@ -11,31 +11,34 @@ tags:
+
+<!-- Setting up [advanced Visibility](/concepts/what-is-advanced-visibility) enables access to creating and using multiple custom Search Attributes with your List Filters. -->
+
 For details, see [Search Attributes](/concepts/what-is-a-search-attribute).
 
-Note that if you use MySQL, PostgreSQL, or SQLite as your Visibility store, Temporal Server version 1.20 and later supports Advanced Visibility features on MySQL (version 8.0.17 and later), PostgreSQL (version 12 and later) and SQLite (v3.31.0 and later), in addition to Elasticsearch.
+Note that if you use MySQL, PostgreSQL, or SQLite as your Visibility store, Temporal Server version 1.20 and later supports advanced Visibility features on MySQL (version 8.0.17 and later), PostgreSQL (version 12 and later) and SQLite (v3.31.0 and later), in addition to Elasticsearch.
 
-To enable Advanced Visibility on your SQL databases, ensure that you do the following:
+To enable advanced Visibility on your SQL databases, ensure that you do the following:
 
-- [Upgrade your Temporal Server](/clusters/how-to-upgrade-the-temporal-server-version) to version 1.20.
+- [Upgrade your Temporal Server](/clusters/how-to-upgrade-the-temporal-server-version) to version 1.20 or later.
 - [Update your database schemas](/clusters/how-to-upgrade-the-temporal-server-version#upgrade-mysql-or-postgresql-schema) for MySQL to version 8.0.17 (or later), PostgreSQL to version 12 (or later), or SQLite to v3.31.0 (or later).
 
+From Temporal Server v1.21 onwards, you can set up a secondary Visibility store in your Temporal Cluster to enable [Dual Visibility](/concepts/what-is-dual-visibility). This is useful for migrating your Visibility store database.
+
 #### Supported databases
 
 The following databases are supported as Visibility stores:
 
 - [MySQL](/clusters/how-to-set-up-mysql-visibility-store) v5.7 and later.
-  Use v8.0.17 (or later) with Temporal Server v1.20 or later for Advanced Visibility capabilities.
+  Use v8.0.17 (or later) with Temporal Server v1.20 or later for advanced Visibility capabilities.
 - [PostgreSQL](/clusters/how-to-set-up-postgresql-visibility-store) v9.6 and later.
-  Use v12 (or later) with Temporal Server v1.20 or later for Advanced Visibility capabilities.
-- [SQLite](/clusters/how-to-set-up-sqlite-visibility-store) v3.31.0 and later for Advanced Visibility capabilities.
-- [Cassandra](/clusters/how-to-set-up-cassandra-visibility-store).
+  Use v12 (or later) with Temporal Server v1.20 or later for advanced Visibility capabilities.
+- [SQLite](/clusters/how-to-set-up-sqlite-visibility-store) v3.31.0 and later for advanced Visibility capabilities.
+- [Cassandra](/clusters/how-to-set-up-cassandra-visibility-store). Support for Cassandra as a Visibility database is deprecated from Temporal Server v1.21 onwards. Check [Server release notes](https://github.com/temporalio/temporal/releases) for updates.
```

Comment:
We also need to add that standard visibility is being deprecated, and therefore, older versions of MySQL and PostgreSQL that doesn't support advanced visibility are also in deprecation process.

Reply:
Added "Since standard Visibility is deprecated from Temporal Server v1.21 onwards, support for older versions MySQL (v5.7 and earlier) will be dropped." and "Since standard Visibility is deprecated from Temporal Server v1.21 onwards, support for older versions of PostgreSQL (v9.6 and earlier) will be dropped."